src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,310 @@
+                      <Input
+                        name="phoneNumber"
+                        type="text"
+                        value={phoneNumber}
+                        onChange={e => {
+                          changePhoneNumber(e.target.value);
+                          handleChange(e);
+                        }}
+                      />
+                    </FormControl>
+                  </Box>
+                  <Box>
+                    <FormControl id="heard-about" flex="1">
+                      <FormLabel>How Did You Hear About Us?</FormLabel>
+                      <Input
+                        name="heardAbout"
+                        type="text"
+                        value={heardAbout}
+                        onChange={e => {
+                          changeHeardAbout(e.target.value);
+                          handleChange(e);
+                        }}
+                      />
+                    </FormControl>
+                  </Box>
+                </Flex>
+              </Flex>
+              <Box>
+                <Checkbox
+                  defaultChecked={false}
+                  onChange={e => {
+                    changeTermsAndConditionsAccepted(e.target.checked);
+                    handleChange(e);
+                  }}
+                >
+                  <Text fontSize="xs">
+                    I HAVE READ THIS RELEASE OF LIABILITY AND FULLY UNDERSTAND ITS{' '}
+                    {
+                      <Link
+                        textDecoration="underline"
+                        onClick={() => {
+                          changeTermsAndConditionsIsOpened(true);
+                        }}
+                      >
+                        TERMS AND CONDITIONS.
+                      </Link>
+                    }
+                  </Text>
+                </Checkbox>
+              </Box>
+              <HStack>
+                <Box>
+                  <Button type="submit" onClick={prevStep} colorScheme="blue">
+                    Back
+                  </Button>
+                </Box>
+                <Box>
+                  <Button
+                    type="submit"
+                    onClick={() => {
+                      if (termsAndConditionsAccepted === false) {
+                        (function () {
+                          changeSubmitAndAcceptedFalse(true);
+                        })();
+                      } else {
+                        handleSubmit();
+                      }
+                    }}
+                    colorScheme="blue"
+                  >
+                    Submit
+                  </Button>
+                </Box>
+              </HStack>
+              <>
+                {submitAndAcceptedFalse ? (
+                  <>
+                    <Alert status="error">
+                      <AlertIcon />
+                      <AlertTitle>You haven't accepted the Terms and Conditions</AlertTitle>
```

Comment:
🚫 **[eslint]** <[react/no-unescaped-entities](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unescaped-entities.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.